### ISSUE TITLE
Unchecking show files fails to hide files after cold start

### ISSUE THREAD

**Issue by emiel** [careteditor/caret#887]
Version 4.0.0-rc23 (4.0.0-rc23)

1. Start Caret
1. Ensure Show Files is enabled (View -> Show Files) -> Show files checked, files are visible
1. Exit Caret
1. Start Caret - files are visible
1. Attempt to hide files (View -> Show Files) -> Show files unchecked, **files are visible**

In the last step the files are still visible even though we've chosen to hide them.

**Comment by astoilkov** [comment]
Hi Emiel,

The checkbox getting unchecked seems like a bug. The other behavior is by design – a little strange I admit. We are changing that in the next version.

In order to hide the files, you need to either click `Show None` or click `Show Files` twice consecutively. The idea is that the first click on `Show Files` focuses the files sidebar.

**Comment by emiel** [comment]
Ah I see. The checkbox getting unchecked made me think that File/Outline/None were individual toggles. 🤦‍♂ They should thus behave like a combo box (only one selected).

Still a bug there I guess. Thanks for the quick reply.

**Comment by mrbbp** [comment]
yes, have the same pb, strange idea of the 3 show item menu (files, outline, none) and 3 differents shortcuts, can't it be a single short cut and a loop in the choice?
